packages/packages/api/src/medical/client/metriport.ts

```diff
@@ -354,16 +354,18 @@ export class MetriportMedicalApi {
   }
 
   /**
-   * Returns a URL that can be used to download the document.
+   * Fetches the document from S3 and sends a presigned URL
```

Comment:
I don't think we should update the description, is essentially what we did before, just with a diff format, right? No need to expose to the client that we need to download from S3

Reply:
Updated